Gemfile.lock

```diff
@@ -113,7 +113,7 @@ GEM
       fog-core
       nokogiri (>= 1.5.11, < 2.0.0)
     formatador (0.2.5)
-    gds-api-adapters (57.4.2)
+    gds-api-adapters (57.5.0)
```

Comment:
As this change has hit master, you can probably just rebase this branch https://github.com/alphagov/content-data-admin/pull/430